MoreLinq/ReverseComparer.cs

```diff
@@ -23,7 +23,7 @@ sealed class ReverseComparer<T> : IComparer<T>
     {
         readonly IComparer<T> _underlying;
 
-        public ReverseComparer(IComparer<T> underlying)
+        public ReverseComparer(IComparer<T>? underlying)
```

Comment:
Nullable is not enabled for this file. Should probably add it before finishing this PR.

Reply:
It's enabled for the whole project:

https://github.com/morelinq/MoreLINQ/blob/54ccb69f972943a0057a5a546cc795e5e704494b/MoreLinq/MoreLinq.csproj#L123